app/code/community/Bolt/Boltpay/Model/BoltOrder.php

```diff
@@ -41,6 +43,13 @@ class Bolt_Boltpay_Model_BoltOrder extends Mage_Core_Model_Abstract
         'awraf',    //https://ecommerce.aheadworks.com/magento-extensions/refer-a-friend.html#magento1
     );
 
+    /**
+     * @var array  list of country codes for which Bolt requires a region
```

Comment:
IIRC, it was magento that was complaining for missing region

Reply:
@daisy1754 Let me clarify the context/motivation for this PR -

Context:
While Bolt nor Magento technically require a Billing Address for an order, certain ERP systems do.  A request came in from Indique to include billing information in all orders.

However, to address 

> IIRC, it was magento that was complaining for missing region

As I remember it, the problem with region was that Bolt required a region on billing while Magento did not for countries like Bermuda where there are no regions.  This was resolved by using the city as the region to send to Bolt.  While this is related to this PR, it is not the same.

Reply:
Hmm I don't think we validate region. I just tried calling our API and was able to create order w/o region:

```
curl -X POST \
  https://api-sandbox.bolt.com/v1/merchant/orders \
  -H 'Content-Type: application/json' \
  -H 'Origin: http://bolt.com' \
  -H 'X-Api-Key: c117fec29f5c1365dacbc62390b5b6f98c0698a812d7015849319f9f7b863064' \
  -H 'X-Nonce: 123456789123' \
  -H 'cache-control: no-cache' \
  -d '{
  "cart": {
    "total_amount": 110,
    "discounts": [{
      "description": "testdiscount",
      "amount": 250
    }],
    "currency": "USD",
    "items": [{
      "name": "Pizza Slice",
      "reference": "",
      "description": "Test description",
      "sku": "1234567890",
      "total_amount": 360,
      "unit_price": 60,
      "currency": "USD",
      "quantity": 6,
      "options": null,
      "tax_amount": 0
    }],
    "billing_address": {
      "street_address1": "Test",
      "street_address2": "",
      "locality": "San Francisco",
      "country": "United States",
      "country_code": "US",
      "first_name": "Kazuki",
      "last_name": "Nishiura",
      "company": null,
      "phone": "1111111111",
      "email": "kazuki@bolt.com"
    }
  }
}'
```

Reply:
@daisy1754 
In the past and in the current documentation, region was required:
https://docs.bolt.com/v1/reference#create-order

Apparently, it is not now, but it seems like this is a mistake because whereas there is only one San Francisco in the US which is in California, there are several Newtown[s] in the US in several states.

Reply:
let's remove comment 'list of country codes for which Bolt requires a region' since it's not true. I agree Bolt should probably add validation on region but as of right now this comment is not true.

Reply:
done.